pkg/operator/ceph/controller/cluster_info.go

```diff
@@ -118,11 +118,13 @@ func CreateOrLoadClusterInfo(clusterdContext *clusterd.Context, context context.
 		}
 		clusterInfo.Context = context
 
-		err = createClusterAccessSecret(clusterdContext.Clientset, namespace, clusterInfo, ownerInfo)
+		err = CreateOrUpdateClusterAccessSecret(clusterdContext.Clientset, namespace, clusterInfo, ownerInfo)
 		if err != nil {
 			return nil, maxMonID, monMapping, err
 		}
 	} else {
+		// TODO:sp98 rotate cluster access secret
```

Comment:
Before we rotate the keys, the mons must be updated to the required version of ceph. Let's consider triggering the rotation in the `postMonStartupActions()` method instead of here.